fast64_internal/f3d/f3d_gbi.py

```diff
@@ -3093,43 +2949,14 @@ def __init__(self, name, x1, y1, x2, y2):
         self.x2 = x2
         self.y2 = y2
 
+    def fields(self):
+        return self.x1, self.y1, self.x2, self.y2
+
     def to_binary(self):
-        return (
-            self.x1.to_bytes(4, "big")
-            + self.y1.to_bytes(4, "big")
-            + self.x2.to_bytes(4, "big")
-            + self.y2.to_bytes(4, "big")
-        )
+        return (a.to_bytes(4, "big") for a in self.fields)
```

Comment:
I think you forgot `@property` on `fields` ? or what did you mean for this to do

Reply:
Yeah this was an oversight. I have changed it to use @property